.circleci/config.yml

```diff
@@ -0,0 +1,41 @@
+# Javascript Node CircleCI 2.1 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-javascript/ for more details
+#
+version: 2.1
+
+orbs:
+  lambda: dosomething/lambda@0.0.3
+
+jobs:
+  # Install dependencies, run tests, and compile for Lambda.
+  build:
+    docker:
+      - image: circleci/node:lts
+      - image: circleci/dynamodb
+    environment:
+      # Our mock DynamoDB server still requires credentials to be sent,
+      # even though it doesn't do anything with them. Hence...
+      AWS_ACCESS_KEY_ID: XXXXXXXXXXXXXXXXXXXX
+      AWS_SECRET_ACCESS_KEY: XXXXXXXXXXXXXAXXXXXXXXXXXXXXXXXXXXXXXXXX
```

Comment:
This tripped me up for a while, since the `circle/dynamodb` image just seems to time out if it gets a request without any credentials (rather than giving us any sort of useful error)! Ugh!